src/krb5/_chpw_message_mit.pyx

```diff
@@ -0,0 +1,50 @@
+import typing
+
+from krb5._exceptions import Krb5Error
+
+from libc.string cimport strlen
+
+from krb5._context cimport Context
+from krb5._krb5_types cimport *
+
+
+cdef extern from "python_krb5.h":
+    krb5_error_code krb5_chpw_message(
+        krb5_context context,
+        const krb5_data *server_string,
+        char **message_out
+    ) nogil
+
+cdef extern from "python_krb5.h":
```

Comment:
```suggestion
```

`cdef extern from ...` can contain multiple defs without having to declare this everytime.

Reply:
done